src/oge/emissions.py

```diff
@@ -691,6 +704,14 @@ def calculate_generator_nox_ef_per_unit_from_boiler_type(gen_fuel_allocated, yea
         )
     )
     if len(missing_nox_efs) > 0:
+        missing_nox_efs.insert(
```

Comment:
We do not need BA codes on this check since it is not a plant-specific validation check